jiant/preprocess.py

```diff
@@ -224,9 +224,9 @@ def _build_vocab(args, tasks, vocab_path: str):
     if args.input_module == "gpt":
         # Add pre-computed BPE vocabulary for OpenAI transformer model.
         add_openai_bpe_vocab(vocab, "openai_bpe")
-    if args.input_module.startswith("bert"):
-        # Add pre-computed BPE vocabulary for BERT model.
-        add_bert_wpm_vocab(vocab, args.input_module)
+    elif args.input_module.startswith("bert") or args.input_module.startswith("xlnet"):
```

Comment:
nit: `args.input_module.startswith("bert") or args.input_module.startswith("xlnet")` is repeated in quite a few places - worth making it a utility function (`_using_pytorch_transformers(args.input_module)`) so it's less error-prone when adding GPT-2 or other future models?

Reply:
Sure. I'm only replacing the instances of this where the semantics actually seems right, though—for the language modeling check above, for example, we're testing whether the model is a bidirectional transformer, not whether it uses this library.